excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/validation/HiddenFieldsValidator.java

```diff
@@ -18,22 +21,22 @@ public Boolean parseComplexTypesHiddenFields(DefinitionDataItem definitionDataIt
                                                  Map<String, DefinitionSheet> definitionSheets) {
         final DefinitionSheet caseEventToFields = definitionSheets.get(SheetName.CASE_EVENT_TO_FIELDS.getName());
         final DefinitionSheet caseFields = definitionSheets.get(SheetName.CASE_FIELD.getName());
-        Optional<DefinitionDataItem> caseField =
+        List<DefinitionDataItem> caseField =
             caseFields.getDataItems().stream().filter(caseFieldDataItem ->
-                definitionDataItem.getId().equals(caseFieldDataItem.getString(ColumnName.FIELD_TYPE))).findFirst();
-        caseField.ifPresent(ddi -> {
+                definitionDataItem.getId().equals(caseFieldDataItem
+                    .getString(ColumnName.FIELD_TYPE))
+                    || definitionDataItem.getId().equals(caseFieldDataItem
+                    .getString(ColumnName.FIELD_TYPE_PARAMETER))).collect(toList());
+
+        validateCaseEventToFields(definitionDataItem, caseEventToFields, caseField);
+
+        caseField.forEach(ddi -> {
             Optional<DefinitionDataItem> caseEventToField = caseEventToFields.getDataItems()
                 .stream().filter(definitionDataItem1 -> ddi.getId()
                     .equals(definitionDataItem1.getCaseFieldId())).findFirst();
             caseEventToField.ifPresent(caseEventToFieldDataItem -> {
-                String fieldShowCondition = caseEventToFieldDataItem.getString(ColumnName.FIELD_SHOW_CONDITION);
                 Boolean caseFieldRetainHiddenValue = caseEventToFieldDataItem.getRetainHiddenValue();
-                if (Boolean.TRUE.equals(isShowConditionPopulated(fieldShowCondition, definitionDataItem))) {
-                    throw new MapperException(String.format("'retainHiddenValue' can only be configured "
-                            + "for a field that uses a "
-                            + "showCondition. Field ['%s'] on ['%s'] does not use a showCondition",
-                        caseEventToFieldDataItem.getCaseFieldId(), SheetName.CASE_EVENT_TO_FIELDS.getName()));
-                } else if (Boolean.TRUE.equals(
+                if (Boolean.TRUE.equals(
```

Comment:
Not directly related to this PR, but in various places it looks like there is `if (Boolean.TRUE.equals(condition))` which could just be `if (condition)` and return a `boolean`